### PR TITLE
After Rebase

### PR DESCRIPTION
**Pull request by damodarS**

### CODE REVIEW

File: MyFirstFile [hunk 1]
```diff
@@ -1,2 +1,3 @@
 Master-line 1
 alex-line2
+john-line2
```
Comment (damodarS) [inline MyFirstFile:3]: request to check both lines

**Review by pratyusha1305 — COMMENTED** [review]
testing . check from u r end also

**Review by damodarS — COMMENTED** [review]
> testing . check from u r end also

will revert-john

File: MyFirstFile (hunk 1 above)
Comment (damodarS) [reply MyFirstFile:3]: simple. dne

**Pull request merged**